src/main/java/org/ihtsdo/buildcloud/config/SecurityContextConfiguration.java

```diff
@@ -5,26 +5,32 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.security.config.annotation.web.builders.HttpSecurity;
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configurers.AbstractHttpConfigurer;
 import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.web.SecurityFilterChain;
 import org.springframework.security.web.access.intercept.FilterSecurityInterceptor;
 
+import static org.springframework.security.config.Customizer.withDefaults;
+
 @Configuration
 @EnableWebSecurity
 public class SecurityContextConfiguration {
 
 	@Bean
 	public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
-		http.sessionManagement().sessionCreationPolicy(SessionCreationPolicy.STATELESS);
-		http.authorizeRequests()
-				.antMatchers("/snomed-release-service-websocket/**/*",
+		http.httpBasic(withDefaults());
+		http.csrf(AbstractHttpConfigurer::disable);
+		http.sessionManagement(sessionManagement -> sessionManagement.sessionCreationPolicy(SessionCreationPolicy.STATELESS));
+		http.addFilterBefore(new RequestHeaderAuthenticationDecorator(), FilterSecurityInterceptor.class);
+		http.authorizeHttpRequests((authorize) -> authorize.requestMatchers(
```

Comment:
During authoring services testing /**/* won't work. You either nee to use ant path matcher or use /**/ only

Reply:
The changes have been made. Thanks @CoderMChu for your help.